content/use-cases/defi/algorithmic-trading.md

```diff
@@ -0,0 +1,89 @@
+
+- Decentralized exchange data is public and freely available. You don't need a subscription or special access to see what trades are available.
+- Trades settle in seconds, which is fast enough that you can do a lot of trading, but not so fast that you need a specialized network connection or extremely powerful computer. You can trade using a laptop and decent home internet connection.
+- Transaction fees on the XRP Ledger are very low, so you don't need much capital to start trading.
+
+## Trading Strategies
+
+Algorithmic trading can make profits through many different strategies; part of the challenge (or the fun) of algorithmic trading is designing and implementing your own unique approach. Other traders may be reluctant to share their strategies in detail, because that would lead to more people competing to take the same opportunities. From a high level, algorithmic trading approaches often fall into the following categories:
+
+- **Arbitrage:** Buying and immediately selling an asset to take advantages of price differences. This could involve finding sets of 3 or more assets where the prices aren't aligned, or using the XRP Ledger to move assets between private exchanges where the prices are different.
+- **Quantitative Trading:** Predicting and taking advantage of future price movements based on past price movements, outside data, or both. Examples include [candlestick patterns](https://blog.quantinsti.com/candlestick-patterns-meaning/), correlating an asset's price movements with other assets, and using sentiment analysis of social media.
+- **Front-running:** Taking advantage of pending trades, especially large ones, by buying and immediately reselling the assets those trades are buying. While the XRP Ledger's canonical ordering of transactions makes front-running difficult, it is not impossible.
+
+## Background Reading
+
+You can familiarize yourself with algorithmic trading, in general, by reading the following resources:
+
+- [Investopedia: Basics of Algorithmic Trading: Concepts and Examples](https://www.investopedia.com/articles/active-trading/101014/basics-algorithmic-trading-concepts-and-examples.asp)
+- [_Flash Boys: A Wall Street Revolt_ by Michael Lewis](https://wwnorton.com/books/Flash-Boys/)
+- [Investopedia: How Arbitraging Works in Investing, With Examples](https://www.investopedia.com/terms/a/arbitrage.asp)
+
+The following pages describe key elements of how the XRP Ledger's decentralized exchange works:
+
+- [Tokens](tokens.html)
+- [Decentralized Exchange](decentralized-exchange.html)
+- [Offers](offers.html)
+
+The following articles provide some more specific examples and interesting information about how these strategies work on other blockchains. This information isn't necessary to get started, but may help to provide perspective.
```

Comment:
You mention this information isn't necessary to get started. Maybe move this into a See Also section at the end?